### ISSUE TITLE
Create settings page

### ISSUE THREAD

**Issue by ArnasDickus** [ArnasDickus/Akademija#50]
Create a settings page.
On Submit you need to check to log in again. Khan academy example.
**Basics:**
1. DisplayName (Slapyvardis) @ when commenting, and what you see on header.
2. Gender -> Male Female Other
3. Birthdate -> Open calendar.
4. Bio

**Change Email**
1. Current email input
2.Enter new email 

**Password:**
1. New Password
2. Reenter a new password

**Delete your account:**
1. Delete your account button.

**Fix bug:**

1. On Courses page profile menu is hidden by video player.

Inspiration
![image](https://user-images.githubusercontent.com/38657595/105285130-759af180-5bbc-11eb-8329-0d31003cb18b.png)

**Comment by ArnasDickus** [comment]
Alright, so it seems my current plan wasn't successful. I was stuck with connecting formik with Material ui. After I succeeded I decided that I want to redo settings page.
Here's what I want to have:
1) Profile:
DisplayName (Slapyvardis) @ when commenting, and what you see on header.
Gender -> Male Female Other
Birthdate -> Open calendar.
Bio
2) Photo -> Allow user to upload photo.
3) Account -> Change Email, Change password.
4) Close Account

To look similiar to udemy
![image](https://user-images.githubusercontent.com/38657595/106418755-b4a03100-645f-11eb-9d0c-657de68c70ab.png)
